jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/WebAppDefaultServletCacheTest.java

```diff
@@ -48,7 +48,7 @@ protected void setUp() throws Exception
     }
 
     @AfterEach
-    protected void tearDown() throws Exception
+    public void tearDown() throws Exception
```

Comment:
See previous comment.

Reply:
I've moved this to PR #10180 (same as above), once that's merged, this one will go away here.